packages/app/src/cli/utilities/developer-platform-client/app-management-client/graphql/list-dev-stores.ts

```diff
@@ -0,0 +1,63 @@
+import {gql} from 'graphql-request'
+
+// eslint-disable-next-line no-warning-comments
+/* TODO for more options like pagination/sort/search see https://github.com/Shopify/business-platform/pull/19676#issuecomment-2165895156 */
```

Comment:
I don't think we need to be able to search for the store. Generally speaking, if someone want to specify a store, they can use the `--store` flag pretty easily. It's come up exceedingly rarely that someone exceeded 50 available stores on an org and got confused, but it was never a high enough priority to really jump in and solve this, because most users in huge orgs do figure this out just fine.

Reply:
I've remove this part of the query, to simplify things